### DISCUSSION TITLE
Few questions about shortcuts in macos

### DISCUSSION THREAD

**Discussion by thetruechar** [kovidgoyal/kitty#3843] [Q&A]
I use the [sample config](https://sw.kovidgoyal.net/kitty/_downloads/433dadebd0bf504f8b008985378086ce/kitty.conf) of
kitty 0.21.2 in macOS 11.4

![image](https://user-images.githubusercontent.com/9455827/126034970-918a7db2-4a85-4564-95c3-3d3e5230fed7.png)
I found that the shortcuts in the menu are not the ones in the config file

and for examples shortcuts defined in the config file do not work, for close_window,  only the `shift-cmd-d` works
`map kitty_mod+w close_windows
` not works

So what's wrong here? thanks.

**Comment by thetruechar** [comment]
I also notice that ctrl+shift+k/j can scroll up/down, but not found these definitions in the config file, why?

**Reply by kovidgoyal** [reply]
why would they be? Why do you think the config file is exhaustive? If you want exhaustive documentation, read https://sw.kovidgoyal.net/kitty/conf.html, IIRC that includes all the platform specific shortcut variants as well. And as I said before kitty_mod+w works fine for me on macOS.

**Reply by thetruechar** [reply]
Sorry for bothering you, I didn't realize that the config file is not exhaustive. The shortcuts do not work is due to the shortcuts conflict with other apps.
However, I have one more question... Can I disable one shortcut, for example, the Close OS Window shift+cmd+w shown in the menu?

**Comment by kovidgoyal** [comment]
On Sat, Jul 17, 2021 at 09:52:32AM -0700, charlie wrote:
> Sorry for bothering you, I didn't realize that the config file is not exhaustive. The shortcuts do not work is due to the shortcuts conflict with other apps.
> However, I have one more question... Can I disable one shortcut, for example, the `Close OS Window` `shift+cmd+w` shown in the menu?

yes you can read the shortcuts section in the conf link I posted earlier
to learn how.

**Reply by thetruechar** [reply]
Thank you

**Reply by b225ccc** [reply]
> yes you can read the shortcuts section in the conf link I posted earlier to learn how.

I want to do the same thing, but I can't find how to override a _single_ shortcut.  I see [clear_all_shortcuts](https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.clear_all_shortcuts), but no way to override a single one.  For example, I'd like to remap `shift+cmd+d` and to disable `cmd+q`.  Any more pointers would be appreciated!  Thanks

**Reply by Luflosi** [reply]
Map <kbd>shift</kbd>+<kbd>cmd</kbd>+<kbd>d</kbd> to the action you want and also map <kbd>cmd</kbd>+<kbd>d</kbd> to `no_op`.

**Reply by b225ccc** [reply]
> Map shift+cmd+d to the action you want and also map cmd+d to `no_op`.

That works - thanks!  (I did try that earlier, but I think I needed a full restart of kitty instead of a config reload.)

Not sure if this is expected, but adding this line to config had no effect:  `map cmd+q no_op`.  `cmd+q` still quit kitty.

But... I discovered the `confirm_os_window_close` option, which solves the problem I had with accidental quit.

Thanks again for your help.

**Reply by Luflosi** [reply]
See if `map kitty_mod+q no_op` works.

**Comment by thetruechar** [comment]
![image](https://user-images.githubusercontent.com/9455827/126045784-7a255349-18bf-437c-8308-9620bf596131.png)
small question: when hit show_scrollback,  what are these `<E2><9E><9C>` mean?
